docs/reference/engine.md

```diff
@@ -70,6 +70,12 @@ source .venv/bin/activate
 pip install --requirement requirements.txt
 ```
 
+Start the dependencies.
+
+```
+docker-compose up
```

Comment:
If you have a recent Docker/Docker Compose installation, you can use the command `docker compose` instead of the now legacy `docker-compose` command: https://docs.docker.com/compose/migrate/

Reply:
Fixed.